src/assets/icons/parent_avatar.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="23" viewBox="0 0 24 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M21.3332 24.5V22.1667C21.3332 19.5893 19.2438 17.5 16.6665 17.5H7.33317C4.75584 17.5 2.6665 19.5893 2.6665 22.1667V24.5" fill="#7B8794"/>
+<path d="M21.3332 24.5V22.1667C21.3332 19.5893 19.2438 17.5 16.6665 17.5H7.33317C4.75584 17.5 2.6665 19.5893 2.6665 22.1667V24.5" stroke="#7B8794" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12.0002 12.8333C14.5775 12.8333 16.6668 10.744 16.6668 8.16667C16.6668 5.58934 14.5775 3.5 12.0002 3.5C9.42283 3.5 7.3335 5.58934 7.3335 8.16667C7.3335 10.744 9.42283 12.8333 12.0002 12.8333Z" fill="#7B8794" stroke="#7B8794" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
Curious to know what's going on in this file - were there some adjustments needed to the avatars in the application in order to have them be housed in the header? Also do parents have Avatars?

Reply:
This SVG is from the Figma doc provided to us by the UX team. At the moment, parents don't necessarily have an avatar, and the name of this SVG is slightly misleading. This icon is only used to represent the parent's account in the navbar.